pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -66,8 +66,18 @@ def circuit(theta, phi):
         grad_def = qml.grad(qnode_def, argnum=[0, 1])
         grad = qml.grad(qnode, argnum=[0, 1])
 
-        assert np.allclose(qnode(theta, phi), qnode_def(theta, phi), atol=tol(dev.shots))
-        assert np.allclose(grad(theta, phi), grad_def(theta, phi), atol=tol(dev.shots))
+        def workload():
+            return (
+                qnode(theta, phi),
+                qnode_def(theta, phi),
+                grad(theta, phi),
+                grad_def(theta, phi),
```

Comment:
do we want to benchmark the tested device as well as the reference device?

Reply:
Good question.
Yes. At least for now, this is the plan.
The selection of benchmarks, and possible the approach to do that, will be refined in the next iteration.